tests/integrationtest/r/ddl/active_active.result

```diff
@@ -56,6 +70,12 @@ create table t5 (id int primary key, name varchar(50)) active_active = 'ON' SOFT
 Error 8200 (HY000): Unsupported ACTIVE_ACTIVE = 'ON' but SOFTDELETE = 'OFF'
 alter table t3 active_active = 'ON';
 Error 8200 (HY000): This type of ALTER TABLE is currently unsupported
+create table t5 (id int primary key, name varchar(50)) active_active = 'ON';
+Error 8200 (HY000): Unsupported ActiveActive enabled but SoftDelete disabled
+create table t5 (id int primary key, name varchar(50)) active_active = 'ON' SOFTDELETE = 'OFF';
+Error 8200 (HY000): Unsupported ACTIVE_ACTIVE = 'ON' but SOFTDELETE = 'OFF'
+alter table t3 active_active = 'ON';
+Error 8200 (HY000): This type of ALTER TABLE is currently unsupported
```

Comment:
There are other limitations:

- A softdelete / active-active table does not support the foreign key (as a child or parent table) 
- A softdelete / active-active table does not support a temporary table